packages/editor-ui/src/components/__tests__/SQLEditor.test.ts

```diff
@@ -0,0 +1,113 @@
+import { render } from '@testing-library/vue';
+import { PiniaVuePlugin } from 'pinia';
+import { SETTINGS_STORE_DEFAULT_STATE, waitAllPromises } from '@/__tests__/utils';
+import { STORES } from '@/constants';
+import { createTestingPinia } from '@pinia/testing';
+import { merge } from 'lodash-es';
+
+import SqlEditor from '@/components/SqlEditor/SqlEditor.vue';
+import { expressionManager } from '@/mixins/expressionManager';
+import type { TargetItem } from '@/Interface';
+
+const RESOLVABLES: { [key: string]: string | number | boolean } = {
+	'{{ $json.schema }}': 'public',
+	'{{ $json.table }}': 'users',
+	'{{ $json.id }}': 'id',
+	'{{ $json.limit - 10 }}': 0,
+	'{{ $json.active }}': false,
+};
+
+const DEFAULT_SETUP = {
+	pinia: createTestingPinia({
+		initialState: {
+			[STORES.SETTINGS]: {
+				settings: merge({}, SETTINGS_STORE_DEFAULT_STATE.settings),
+			},
+		},
+	}),
+	props: {
+		dialect: 'PostgreSQL',
+		isReadOnly: false,
+	},
+};
+
+const renderComponent = (renderOptions: Parameters<typeof render>[1] = {}) =>
+	render(SqlEditor, merge(DEFAULT_SETUP, renderOptions), (vue) => {
+		vue.use(PiniaVuePlugin);
+	});
+
+describe('SQL Editor Preview Tests', () => {
```

Comment:
Since the test filename shows up in the test run report, I'm never sure if we should write a `describe` block for a single-block file.

Reply:
Agree, but I like to put a descriptive message here.